app_server/app/models/Storage.js

```diff
@@ -164,4 +170,24 @@ Class.getListOfUsers = function() {
     });
 };
 
+/**
+ * Check if the fields to be changed match the fields available in object
+ * @private
+ */
+var _isFieldsMatch = function(user, options, fn) {
+  var fieldsToChange = options.fields;
+  var index = fieldsToChange.indexOf('updatedAt');
+
+  if (index > -1) {
+    fieldsToChange.splice(index, 1);
+  }
+
```

Comment:
This function can be define as `function isFieldsMatched`, the `_` is not needed to mark privacy since this function will not be exported.